cmd/operator/main.go

```diff
@@ -20,11 +20,11 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/kubeflow/spark-operator/cmd/operator"
+	"github.com/kubeflow/spark-operator/cmd/operator/app"
 )
 
 func main() {
-	if err := operator.NewCommand().Execute(); err != nil {
+	if err := app.NewCommand().Execute(); err != nil {
```

Comment:
nit: I don't think the `app` package/folder is strictly necessary as the `cobra.Command` structure could be initialised in this file.

Reply:
Change LGTM besides this small nit

Reply:
Done.